models/cart.model.js

```diff
@@ -0,0 +1,12 @@
+const mongoose = require("mongoose")
+const { Schema } = mongoose;
+
+const { Product } = require('./product.model');
+
+const cartSchema = new Schema({
+  product: { type: Schema.Types.ObjectId, ref: 'Product' },
+  quantity: Number
+});
```

Comment:
You should include a timestamp.

Reply:
I dint want to add it because I am not showing it on the front end right?